spring-cloud-azure-samples/spring-cloud-azure-cache-sample/README.adoc

```diff
@@ -27,12 +27,15 @@ This starter is based on spring-boot-starter-data-redis. It fetches and auto con
 1.  Create azure credential file. Please see https://github.com/Azure/azure-libraries-for-java/blob/master/AUTH.md[how to create credential file]
 +
 ....
-$  az az account set --subscription <name or id>
+$  az account set --subscription <name or id>
 $  az ad sp create-for-rbac --sdk-auth > my.azureauth
 ....
++
+Make sure `my.azureauth` is encoding with UTF-8.
+
 2.  Put auth file under `src/main/resources/`
-3.  Create Azure redis cache instance through Azure Portal or CLI. Please see https://azure.microsoft.com/en-us/services/cache/[How to create Azure redis cache]
```

Comment:
We need to keep the link about how to create redis, rather than just show the portal

Reply:
will update for different resource creation.